egrismore/Lab4.py

```diff
@@ -0,0 +1,27 @@
+__author__ = 'emilygrismore'
+"""checks requirements for and runs the magic 1089"""
+
+
+def flip(i):
+    """reverses the number"""
+    int(str(i)[::-1])
+    return i
+
+
+def decrease(i):
+    """checks to see if the #'s are in descending order"""
+    largest = 10
+    for new in str(i):
+        number = int(new)
+        if number >= largest:
+            return False
+        else:
+            largest = number
+    return True
+
+
+def trick(i):
+    """shows the magical 1089"""
+    difference = i - flip(i)
+    difference + flip(difference)
+    print(i)
```

Comment:
Yeah, I worked on it a lot yesterday. I expected it to be easy but for
some reason I keep getting an assertion error. I am almost certain I am
doing right but it won't pass the nose tests. It is driving me slightly
crazy.

On Tue, Oct 13, 2015 at 12:15 PM, Kyle Tarplee <notifications@github.com>
wrote:

> In egrismore/Lab4.py
> <https://github.com/AndersonUniversity/CPSC4500test/pull/3#discussion_r41888621>
> :
>
> > +    """checks to see if the #'s are in descending order"""
> > +    largest = 10
> > +    for new in str(i):
> > +        number = int(new)
> > +        if number >= largest:
> > +            return False
> > +        else:
> > +            largest = number
> > +    return True
> > +
> > +
> > +def trick(i):
> > +    """shows the magical 1089"""
> > +    difference = i - flip(i)
> > +    difference + flip(difference)
> > +    print(i)
>
> Emily, are you working on fixing this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AndersonUniversity/CPSC4500test/pull/3/files#r41888621>
> .
>